### PR TITLE
WRP-4194: Apply correct lint option to samples

### PR DESCRIPTION
**Pull request by andreitirlalge**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Some samples did not have the correct eslint configuration.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Applied the correct lint option to samples and fixed eslint warnings that occured after this change.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Due to some changes made in the configuration of the eslint, new warnings occured that were fixed in this PR as well.
(e.g ImportSkins.js, utils.js, SamplePage.js, Content.js)

### Links
[//]: # (Related issues, references)
WRP-4194

### Comments
Enact-DCO-1.0-Signed-off-by: Andrei Tirla andrei.tirla@lgepartner.com

### CODE REVIEW

File: sandstone/custom-skin/src/components/ImportSkin/ImportSkin.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ const ImportSkin = kind({
 							try {
 								reader.readAsText(inputEvent.target.files[0]);
 							} catch (err) {
-								console.log(err);
+								console.log(err);	//eslint-disable-line
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/components/ImportSkin/ImportSkin.js:91]: /* eslint-disable no-console */   at the beggining of the file

File: sandstone/pattern-react18-new/src/views/Suspense/SamplePage.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import {VirtualGridList} from '@enact/sandstone/VirtualList';
 import ri from '@enact/ui/resolution';
 
 const items = [];
-const renderItem = ({index, ...rest}) => {
+const renderItem = ({index, ...rest}) => {	//eslint-disable-line
```
Comment (daniel-stoian-lgp) [inline sandstone/pattern-react18-new/src/views/Suspense/SamplePage.js:7]: Please actually solve the eslint warning. there's no point in suppressing it

renderItem.propTypes = {
	index: PropTypes.number,
};

File: sandstone/custom-skin/src/utils.js [hunk 3]
```diff
@@ -188,7 +188,7 @@ const getColorsFromString = (colors) => {
 
 		return colorSets;
 	} catch (err) {
-		console.log(err);
+		console.log(err);	//eslint-disable-line
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/utils.js:191]: /* eslint-disable no-console */   at the beggining of the file

File: sandstone/pattern-react18-new/src/views/UseTransition/Content.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ const CONTENT = {
 	Three: 'This is the content of the third page'
 };
 
-function Content ({page}) {
+function Content ({page}) {		//eslint-disable-line
```
Comment (daniel-stoian-lgp) [inline sandstone/pattern-react18-new/src/views/UseTransition/Content.js:13]: Please actually solve the eslint warning. There's no point in suppressing it

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by seunghoh — CHANGES_REQUESTED** [review]
Please use the PR template even in the samples Repo. 
I see some other code fixes beyond the lint option. Then you should mention it in the PR description for the code reviewer.
CC: @daniel-stoian-lgp

**Comment by andreitirlalge** [comment]
I've added a PR template and changed the description.
CC: @seunghoh

**Review by seunghoh — APPROVED** [review]

**Comment by hong6316** [comment]
to @andreitirlalge 

1. Item "Resolution" is empty.
Please fill out the appropriate content.
(I think it's okay to copy the contents of "Issue Resolved/Feature Added")

2. When checking PR, some changes are different from the main purpose of this PR (e.g. Import Skin.js, utils.js, etc.).
Please write these changes in "Additional Considers".

**Review by hong6316 — APPROVED** [review]
LGTM

**Pull request merged**